RecoTracker/MkFitCore/src/Matriplex/Matriplex.h

```diff
@@ -182,11 +300,180 @@ namespace Matriplex {
   using MPlex = Matriplex<T, D1, D2, N>;
 
+  //==============================================================================
+  // Operators
+  //==============================================================================
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator+(const MPlex<T, D1, D2, N>& a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t += b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator-(const MPlex<T, D1, D2, N>& a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t -= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator*(const MPlex<T, D1, D2, N>& a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t *= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator/(const MPlex<T, D1, D2, N>& a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t /= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator+(const MPlex<T, D1, D2, N>& a, T b) {
+    MPlex<T, D1, D2, N> t = a;
+    t += b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator-(const MPlex<T, D1, D2, N>& a, T b) {
+    MPlex<T, D1, D2, N> t = a;
+    t -= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator*(const MPlex<T, D1, D2, N>& a, T b) {
+    MPlex<T, D1, D2, N> t = a;
+    t *= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator/(const MPlex<T, D1, D2, N>& a, T b) {
+    MPlex<T, D1, D2, N> t = a;
+    t /= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator+(T a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t += b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator-(T a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t -= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator*(T a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t *= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> operator/(T a, const MPlex<T, D1, D2, N>& b) {
+    MPlex<T, D1, D2, N> t = a;
+    t /= b;
+    return t;
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> sqrt(const MPlex<T, D1, D2, N>& a) {
+    MPlex<T, D1, D2, N> t;
+    return t.sqrt(a);
+  }
+
+  template <typename T, idx_t D1, idx_t D2, idx_t N>
+  MPlex<T, D1, D2, N> sqr(const MPlex<T, D1, D2, N>& a) {
+    MPlex<T, D1, D2, N> t;
+    return t.sqrt(a);
```

Comment:
Pretty sure there's a typo here? Seems like it should be `return t.sqr(a);`

Reply:
Thank you Steve, I added this change into prop-to-plane PR, cms-sw/cmssw#43146, commit ab89b2f412d3f6120c1ca18e4aa54cbef15c0555